products/centreon.md

```diff
@@ -0,0 +1,134 @@
+releases:
+-   releaseCycle: '23.10'
+    releaseDate: 2023-10-30
+    support: 2024-10-31
+    eol: 2025-10-31
+    latest: '23.10.10'
+    latestReleaseDate: 2024-03-14
+
+-   releaseCycle: '23.04'
+    releaseDate: 2023-04-26
+    support: 2024-04-30
+    eol: 2025-04-30
+    latest: '23.04.16'
+    latestReleaseDate: 2024-03-14
+
+-   releaseCycle: '22.10'
+    releaseDate: 2022-10-26
+    support: 2023-10-31
+    eol: 2024-10-31
+    latest: '22.10.20'
+    latestReleaseDate: 2024-03-14
+
+-   releaseCycle: '22.04'
+    releaseDate: 2022-05-18
+    support: 2023-05-31
+    eol: 2024-05-31
+    latest: '22.04.22'
+    latestReleaseDate: 2024-03-14
+    link: "https://docs.centreon.com/docs/__RELEASE_CYCLE__/releases/centreon-core/#{{'__LATEST__'|replace:'.',''}}"
+
+-   releaseCycle: '21.10'
+    releaseDate: 2021-11-02
+    support: 2022-11-30
+    eol: 2023-11-30
+    latest: '21.10.17'
+    latestReleaseDate: 2023-11-17
+    link: https://archives-docs.centreon.com/21.10/docs/releases/centreon-core/#211017
+
+# 21.04 and lower were supported for 18 months (support(x) = eol(x) - 1 year).
+-   releaseCycle: '21.04'
+    releaseDate: 2021-04-21
+    support: 2021-10-31
+    eol: 2022-10-31
+    latest: '21.04.20'
+    latestReleaseDate: 2022-10-12
+    link: https://archives-docs.centreon.com/21.04/docs/releases/centreon-core/#210420
+
+-   releaseCycle: '20.10'
+    releaseDate: 2020-10-19
+    support: 2021-05-31
+    eol: 2022-05-31
+    latest: '20.10.18'
+    latestReleaseDate: 2022-06-09
+    link: https://archives-docs.centreon.com/20.10/docs/releases/centreon-core/#201018
+
+-   releaseCycle: '20.04'
+    releaseDate: 2020-04-20
+    support: 2020-10-31
+    eol: 2021-10-31
+    latest: '20.04.20'
+    latestReleaseDate: 2021-11-19
+    link: https://archives-docs.centreon.com/20.04/docs/releases/centreon-core/#200420
+
+-   releaseCycle: '19.10'
+    releaseDate: 2019-10-15
+    support: 2020-04-30
+    eol: 2021-04-30
+    latest: '19.10.23'
+    latestReleaseDate: 2021-05-04
+    link: https://docs-older.centreon.com/docs/centreon/en/latest/release_notes/centreon-19.10.html
+
+-   releaseCycle: '19.04'
+    releaseDate: 2019-04-24
+    support: 2019-10-31
+    eol: 2020-10-31
+    latest: '19.04.20'
+    latestReleaseDate: 2020-11-19
+    link: https://docs-older.centreon.com/docs/centreon/en/latest/release_notes/centreon-19.04.html
+
+-   releaseCycle: '18.10'
+    releaseDate: 2018-10-24
+    support: 2019-04-30
+    eol: 2020-04-30
+    latest: '18.10.12'
+    latestReleaseDate: 2020-02-24
+    link: https://docs-older.centreon.com/docs/centreon/en/latest/release_notes/centreon-18.10.html
+
+---
+
+> [Centreon](https://www.centreon.com/) is a software platform used to monitor IT infrastructure.
+> It can track the health of systems, applications, and networks, notifying IT staff of any problems.
+> Centreon has a free, open-source core and offers paid modules for additional features.
```

Comment:
```suggestion
```

We don't call out other open-core stuff, and we track [almost all the examples](https://en.wikipedia.org/wiki/Open-core_model) on Wikipedia. I think it is okay to not mention this (and avoid getting into corrections about licensing, whether something is open-core or open-source) across all pages.